### ISSUE TITLE
Better way to control Spark availability

### ISSUE THREAD

**Issue by jonathanindig** [polynote/polynote#739]
Right now if you want to launch a Spark-enabled interpreter you need to have some Spark configuration in the notebook. We should just have a checkbox that toggles whether to launch Spark or not.

**Comment by antonkulaga** [comment]
@jonathanindig the other problem is that if you forgot to mention spark.master and have spark dependencies in the dependency list it does not recognize org.apache.spark and does not explain why. It creates a lot of frustration as it should somehow inform the user.

**Comment by jonathanindig** [comment]
@antonkulaga Agreed, we could definitely make Spark availability for a given session much more obvious than it is now. Relying on configuration presence is definitely unintuitive. 

> have spark dependencies in the dependency list

Out of curiosity, what dependencies did you have in your dependency list?
